statefun-kafka-io/src/test/java/org/apache/flink/statefun/sdk/kafka/testutils/Matchers.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.statefun.sdk.kafka.testutils;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import org.apache.flink.statefun.sdk.kafka.KafkaIngressStartupPosition;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class Matchers {
+
+  public static Matcher<KafkaIngressStartupPosition> startupPositionOf(
```

Comment:
👍 For using a custom matcher!
I'm wondering if it actually make senes to add `.equals()` to startup position, since they equality is well defined anyways.

Reply:
Good point, will change.